notebooks/220-cross-lingual-books-alignment/220-cross-lingual-books-alignment.ipynb

```diff
@@ -0,0 +1,1231 @@
9bFnzx5cunQJDg4O2LhxI7y8vOSuzNyrVy+cOXMGFy5cQHp6OsqVK4dRo0blOvn48P3S0dFBhQoVMHr0aIX1Vd3/tbS0sH79eqxevRqHDx/GyZMnYWpqCicnJ4Ubqn4NgwYNQmpqKg4dOoSjR4+iZs2aWLt2LZYsWZKvdmfMmIGUlBRMmjQJJUqU+OpJHBUMDaEwxh+JiL4BiYmJ+P777zFq1CgMHjy4sMMh+mZxzg4RkQq8e/dOoSx7DlGdOnW+djhE9AEexiIiUoGjR4/iwIEDaNiwIQwNDXHjxg0cPnwY7u7u4u1FiKhwMNkhIlIBqVQKLS0tbNiwAcnJyTA3N4e3tzdGjRpV2KERffM4Z4eIiIjUGufsEBERkVpjskNERERqjckOfXVXrlyBVCrN8a7c/fv3x+TJk/Pc9ujRozFy5Mj8hFekTJ8+HX369Pmq21y8eLFSty3o1auXwt3HiYiKEiY736D9+/dDKpXi9u3bSq+bmpqKFStW5Jio5Nf169dx4cIF9O/fXyzLToyyHzVq1ED9+vUxYsQIhIaGKrTRv39/nDx5Evfv38/1drOysrBv3z706tULderUga2tLZo2bQofH588vUaqEh4ejr1792LgwIEA3icVH74Wn3qsWLECANC0aVNIpVL8/PPPOba/e/ducZ0P97N37964f/8+Tp8+nefYY2Ji4OPjg/r168Pe3h6dOnXCsWPHcr1+cHAwpk+fjs6dO6NWrVpfvGjdnj170Lp1a9jZ2cHDw+OTF0o8cuQIOnXqBDs7O9SrVw+TJk3K8bYab9++xcKFC+Hh4QF7e3s0adIEkyZNkrvlRLaoqCiMHDkSLi4ucHJywuDBgxEeHq4Qn1QqxYEDBxTWv3nzJqpXr44FCxaIZZ97r1u1avXZ1yJbUlISfH190b59ezg6OsLe3h5t27bFokWL5G6pMHHiRLn2nZyc0L59e2zatEnuFhYTJ06Eo6PjJ7fn6OiIiRMn5iq2nKSlpeV425PcePfuHSZNmoS2bdvC2dkZjo6OaN++PbZs2ZLrNmUyGdavX4+mTZvCzs4O7dq1w+HDh3OsGxoain79+sHR0RF16tTBuHHjPnl7ltjYWCxevBjt2rWDo6Mj7Ozs0KJFC/j4+CAoKEiu7pc+m3v16oW2bdvman9IHs/GIqWkpqbC19cXw4YNQ926dVXa9saNG1G/fn1UqlRJYVmvXr1gZ2eHzMxMPHjwAP7+/rhy5QoOHz4sd8fvmjVrwtbWFps2bcLChQu/uM13795h2LBhOHfuHL7//nsMHDgQJiYmePHiBY4dO4YDBw7gn3/+KZS7bW/duhXW1taoV68egPdXjO3atau4/Pbt29i2bRsGDRokdzPMDxMDPT09XLlyBdHR0Qp3Rj906BD09PQUbhVhaWmJZs2aYdOmTWjWrNkX49y4caPc86SkJHh5eSEmJgbe3t6wtLTEsWPHMGrUKGRmZqJdu3ZfbPPs2bPYu3cvJBIJypcvL94wNCf+/v6YNm0aWrZsiT59+iAoKEi8rciHN3TcuXMnZsyYgfr162PixImIiorC1q1bERISgj179kBPTw/A+y+9Pn36IDQ0FN27d0eVKlXw/Plz7Ny5E+fPn8fRo0dRsmRJAEBycjK8vb3x9u1bDBw4EDo6Oti8eTN69uyJgIAA8YaUXbt2xYEDB7BgwQI0btxYLM/IyMDUqVNRtmxZDB8+XG6/ypQpg19//VVhf42MjL74+oWHh+Pnn3/Gq1ev0KpVK/z444/Q0dHBgwcPsHfvXgQGBuLEiRNifV1dXcyePRvA+0TvxIkTWLBgAW7fvo2lS5d+cXt5FRwcjG3btuHChQuIjY2FhoYGSpcujRYtWqBXr145fhbk5N27d3j8+DEaNmwIa2traGpq4ubNm+KtLXJzVeWlS5di3bp1+OGHH2BnZ4fTp09jzJgx0NDQgKenp1gvMjISPXr0gJGREUaPHo2UlBRs2rQJDx8+xJ49e+Sulh0cHIwBAwYgOTkZnp6e+Omnn6CrqyvetmP//v3Yvn07vv/+e+VfPFKOQN+cffv2CRKJRAgODlZ63djYWEEikQjLly/P8/YvX74sSCQS4fLly2JZTEyMULNmTWH37t051j127Jhc+c6dOwWJRCKsW7dOof2NGzcKDg4OQlJS0hdjmTFjhiCRSAQ/Pz+FZZmZmcKGDRuEV69e5XLPPi05OVmp+unp6ULdunWFpUuXfrLOsWPHFF7HDzVp0kTo3bu34OTkJGzevFlu2atXr4Tq1asLw4cPz7EvHD9+XJBKpUJYWJhScQuCIKxfv16QSCTCxYsXxbKsrCyhS5cugpubm5CWlvbFNqKjo4XU1FRBEP73HuUkNTVVqFOnjjBgwAC58jFjxggODg5CfHy8IAiCkJaWJri4uAg9evQQZDKZWO/MmTOCRCIRtm7dKpZdv35dkEgkwvbt2+Xa3Lt3ryCRSISTJ0+KZevWrRMkEolw69Ytsezx48dCjRo1hCVLlsit/+jRI6FWrVrCxIkTxbK1a9cKEolEOH36tFzdnj17Cp6enp9+gT4jIyNDaN++vVC7dm3h2rVrCsvfvn0r/P777+LzCRMmCA4ODnJ1srKyhM6dOwsSiUSIjIz8ZL0POTg4CBMmTMh1jDNmzBCkUqng5eUlbNy4UThz5oxw4sQJYc2aNUKHDh0EOzs7hfdAWTNnzhQkEonw+vXrz9aLjIwUatWqJcyYMUMsk8lkgpeXl9CwYUMhMzNTLJ82bZpgb28vvHjxQiy7cOGCIJFIBH9/f7EsPj5ecHNzE9zc3ITHjx8rbFMmkwmHDh2S6ztf+mzOT7/41vEwFgH43xB1VFQUhgwZAkdHR9SrVw8LFixAVlYWACAiIgL169cHAPj6+iocNgHeD++OGDECderUgZ2dHTp37pyrwyH//PMPMjMz4erqmqt4XVxcAEDhcAEAuLq6IiUlBRcvXvxsG5GRkdi1axfc3NxyPNSjpaWFfv36yY3q
```

Comment:
I don't know whether it's common for notebooks but I'd like to have some kind of conclusion like we showed on the optimization step:
On an Intel Core i9-10980XE CPU, the PyTorch model processed X sentences per second with (what is x1.00?). After optimization with OpenVINO in Sync mode.... Using Async mode with throughput hint, we get X boost....
And if we have similar NLP notebooks, we could add next steps with links to docs
 Explore more tutorials/Install OpenVINO/ See Performance Benchmarks

Reply:
Done